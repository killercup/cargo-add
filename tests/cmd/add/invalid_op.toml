bin.name = "cargo-add"
args = ["add", "my-package", "--upgrade=an_invalid_string"]
status.code = 2
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
