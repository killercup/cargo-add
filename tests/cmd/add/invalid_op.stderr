error: "an_invalid_string" isn't a valid value for '--upgrade <METHOD>'
	[possible values: all, default, minor, none, patch]

USAGE:
    cargo add --upgrade <METHOD> <CRATE>...

For more information try --help
